src/spikeinterface/core/basesorting.py

```diff
@@ -190,31 +215,43 @@ def get_times(self, segment_index=None):
         else:
             return None
 
-    def _save(self, format="npz", **save_kwargs):
+    def _save(self, format="numpy_folder", **save_kwargs):
```

Comment:
I think `npz_folder` and `numpy_folder` are pretty obscure references. Probably it will be a good idea to add clarification of pros and cons in the docstring.

Also, an annoying thing of the pattern we are using right now where extractor.save is called from the base class is that this string will not propagate to that method.

Reply:
Short name is fine if it is documented where it is used. right now we don't have documentation of these two formats anywhere do we?